build-system/pr-check/build-targets.js

```diff
@@ -28,14 +28,39 @@ const {gitDiffNameOnlyMaster} = require('../common/git');
 const {isTravisBuild} = require('../common/travis');
 
 /**
- * Checks if the given file is an OWNERS file
+ * Checks if the given file is an OWNERS file.
+ *
  * @param {string} file
  * @return {boolean}
  */
 function isOwnersFile(file) {
   return file.endsWith('OWNERS');
 }
 
+/**
+ * Checks if the given file is a validator protoascii file.
+ *
+ * @param {string} file
+ * @return {boolean}
+ */
+function isValidatorProtoasciiFile(file) {
+  if (!file.startsWith('extensions/')) {
+    return false; // All protoascii files live in extensions/
```

Comment:
We have two other protoascii files that live in validator/: [validator-main.protoascii](https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii) and [validator-css.protoascii](https://github.com/ampproject/amphtml/blob/master/validator/validator-css.protoascii)

Reply:
Those files are already included since they live in `validator/`. I've simplified this logic to run validator checks when any file of the form `validator-*.[html|out|protoascii]` is edited by a PR.

Reply:
Okay, it was the comment that threw me off since it says `All protoascii files live in extensions/`